runtime/syntax/PowerShell.yaml

```diff
@@ -5,7 +5,7 @@ filetype: powershell
 
 detect:
     filename: "\\.ps(1|m1|d1)$"
-    #signature: ""
+    #header: ""
```

Comment:
I suggest to remove this deactivated and thus unneeded line. If it's needed the line must be touched anyway.